repos/fdbt-site/cypress_tests/cypress/support/e2e.ts

```diff
@@ -15,6 +15,8 @@ before(() => {
     addTestTimeRestrictions();
     clickElementByText('Fare day end');
     addTestFareDayEnd();
+    clickElementByText('Operator groups');
+    addTestOperatorGroups();
     cy.log('Global Settings set up for BLAC');
```

Comment:
Not code you have changed, but am I right in saying this should say LNUD? That's what we use now right?

Reply:
let me confirm

Reply:
yes it should changing it